launcher/modplatform/ResourceAPI.h

```diff
@@ -54,7 +54,7 @@ class ResourceAPI {
    public:
     virtual ~ResourceAPI() = default;
 
-    enum ModLoaderType { Forge = 1 << 0, Cauldron = 1 << 1, LiteLoader = 1 << 2, Fabric = 1 << 3, Quilt = 1 << 4 };
+    enum ModLoaderType { Forge = 1 << 0, Cauldron = 1 << 1, LiteLoader = 1 << 2, Fabric = 1 << 3, Quilt = 1 << 4, NeoForge = 1 << 5 };
```

Comment:
Why can't this be reshuffled?

Reply:
if it's not serialized numerically anywhere it can be.

Reply:
so i'm guessing it can be

Reply:
It doesn't look like we serialize it numerically(from a quick search in the code).